qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -119,7 +121,7 @@ def __init__(self, *, win_id, private, parent=None):
         self._win_id = win_id
         self._tab_insert_idx_left = 0
         self._tab_insert_idx_right = -1
-        self.shutting_down = False
+        self.is_shutting_down = False
```

Comment:
Note to self: I need to check how that conflicts with other PRs (merged in the meantime or still open) which use `shutting_down` maybe.